packages/quantic/force-app/main/default/lwc/quanticResultQuickview/__tests__/quanticResultQuickview.test.js

```diff
@@ -0,0 +1,171 @@
+import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
+// @ts-ignore
```

Comment:
```suggestion
```

Reply:
We do need it cause createElement is not recognized.

Reply:
I tought etienne brought it up with salesforce as a bug, it will eventually be fixed but in the meantime (at least on my end it does not seem to prevent the tests from running correctly.

Also dont know about you, but even with the `ts-ignore`, I still get the red line (even restarted the ts server in case) 
<img width="310" alt="image" src="https://github.com/coveo/ui-kit/assets/73316533/b001283c-0949-41cb-9861-d7d6c6536865">